### PR TITLE
[READY] NPM Module

### PR DESCRIPTION
**Pull request by SethNute**
screenshot from 2015-06-18 11 46 17

Above are the files that will be included in the node module via <code>npm install canon-react</code>, dist contains the min version of canon-react and demo contains all the files needed for index.html to display properly (using components defined in canon-react.min.js). 

How to use:
Install via <code>npm install canon-react</code>
Where you would like to use react components from canon-react, add <code>var componentName = require('canon-react').componentName;</code>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,12 @@
   "name": "canon-react",
   "version": "0.1.0",
   "description": "Rackspace Canon components built with React",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rackerlabs/canon-react"
+  },
```
Comment (SethNute) [inline package.json]: Rackerlabs is all open source stuff, right?

Comment (bradgignac) [inline package.json]: There is also some private stuff.

File: src/Button.jsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+var React = require('react');
```
Comment (SethNute) [inline src/Button.jsx]: I understand that here node looks for the package react in a few different places (global and local modules). How was the symbol React being provided globally before adding these require statements?

File: package.json (hunk 1 above)
Comment (SethNute) [inline package.json]: So is this okay to include or no?

**Comment by SethNute** [comment]
Can I go ahead and <code>npm publish</code> for testing purposes? I've tested this by installing it globally on my local machine and requiring it from the node terminal so far.

File: package.json (hunk 1 above)
Comment (adampickeral) [inline package.json]: the private things are actually in private repos, so this should be fine. @bradgignac can correct me if i'm wrong

Comment (bradgignac) [inline package.json]: Oh yeah. It's fine.

**Comment by adampickeral** [comment]
looks pretty good to me, give it a shot!

**Comment by adampickeral** [comment]
:+1: :exclamation:

**Comment by adampickeral** [comment]
:shipit:

**Pull request merged**